toolchain/sem_ir/inst_namer.cpp

```diff
@@ -416,6 +416,11 @@ auto InstNamer::CollectNamesInBlock(ScopeId scope_id,
             sem_ir_.entity_names().Get(inst.entity_name_id).name_id);
         continue;
       }
+      case CARBON_KIND(BindingPattern inst): {
+        // See comment on AddrPattern
```

Comment:
```suggestion
        // See comment on AddrPattern.
```

Please try to keep punctuation on comments.

That said, is this referring to the TODO? Should this be a TODO as well?

```suggestion
        // TODO: See comment on AddrPattern.
```